embassy-stm32/src/rcc/f4.rs

```diff
@@ -28,11 +28,65 @@ pub struct Config {
     pub sys_ck: Option<Hertz>,
     pub pclk1: Option<Hertz>,
     pub pclk2: Option<Hertz>,
+    pub plli2s: Option<Hertz>,
```

Comment:
In chips without i2s pll, this field is still present, but does nothing. That might be confusing for the user. Could you change it to put the `cfg` in the field?